modules/cli/pages/gateway.adoc

```diff
@@ -0,0 +1,352 @@
+
+For example:
+[source, bash]
+----
+$ skupper gateway generate-bundle database.yaml ./
+----
+
+This bundle contains the gateway definition YAML and a  certificate that allow access to the {service-network}.
+
+--
+
+. Copy the gateway definition file, for example, `mylaptop-jdoe.tar.gz` to the machine that hosts the service you want to expose on the {service-network}.
+
+. From the machine that hosts the service you want to expose:
++
+--
+[source, bash]
+----
+$ mkdir gateway
+
+$ tar -xvf <gateway-definition-file> --directory gateway
+$ cd gateway
+$ sh ./launch.py
+----
+
+NOTE: Use `./launch.py -t podman` or `./launch.py -t docker` to run the Skupper router in a container.
+
+Running the gateway bundle uses the gateway definition YAML and a certificate to access and expose the service on the {service-network}.
+
+--
+
+. Check the status of the gateway service:
++
+--
+
+To check a _service_ type gateway:
+[source, bash]
+----
+$ systemctl --user status <gateway-definition-name>
+----
+
+To check a _podman_ type gateway:
+[source, bash]
+----
+$ podman inspect
+----
+
+To check a _docker_ type gateway:
+[source, bash]
+----
+$ docker inspect
+----
+
+
+NOTE: You can later remove the gateway using `./remove.py`.
+
+--
+
+. From the machine with cluster access, check the status of Skupper gateways:
++
+--
+[subs=attributes+]
+----
+$ skupper gateway status
+Gateway Definition:
+╰─ machine-user type:service version:{service-version}
+   ╰─ Bindings:
+      ╰─ mydb:3306 tcp mydb:3306 localhost 3306
+----
+This shows that there is only one exposed service and that service is only exposing a single port (BIND). There are no ports forwarded to the local host.
+--
+
+NOTE: If you need to change the gateway definition, for example to change port, you need to remove the existing gateway and repeat this procedure from the start to redefine the gateway.
+
+// Type: procedure
+[id="gateway-reference"] 
+== Gateway YAML reference
+
+The xref:exporting-gateway[] describes how to create a gateway to apply on a separate machine using a gateway definition YAML file.
+
+The following are valid entries in a gateway definition YAML file.
+
+name:: Name of gateway
+bindings.name:: Name of binding for a single host.
+bindings.host:: Hostname of local service.
+bindings.service:: Definition of service you want to be available on service network.
+bindings.service.address:: Address on the service network, name and port.
+bindings.service.protocol:: Skupper protocol, `tcp`, `http` or `http2`.
+bindings.service.ports:: A single port that becomes available on the service network. 
+bindings.service.exposeIngress:: (optional) The ingress mode for the service.
+bindings.service.tlscredentials:: (optional) The TLS certificate and key for the service.
+bindings.service.tlscertauthority:: (optional) The TLS public certificate.
+bindings.service.publishnotreadyaddresses:: Specify whether to sync services relating to Kubernetes objects scheduled but not yet available.
```

Comment:
@ajssmith Is this ok?

btw just the lines between my comments need to be reviewed

Reply:
Suggest that publishnotreadyaddresses be omitted as gateway does not participate in service sync